pkg/signup/signup_handler.go

```diff
@@ -22,13 +23,13 @@ func NewSignupService(logger *log.Logger, config *configuration.Registry) *Signu
 	}
 }
 
-// HandleRequest returns a default heath check result.
-func (srv *SignupService) HandleRequest(ctx *gin.Context) {
+// CreateSignupHandler returns signup info.
+func (srv *SignupService) CreateSignupHandler(ctx *gin.Context) {
```

Comment:
I would leave the name of this function as it was.  It's not creating a signup handler here, it's handling the signup request (the handler was already *created* previously).

Reply:
it's not about the handler, it's about the resource.. the resource is a POST, which would be to create... see design doc here: https://docs.google.com/document/d/19UzaXVqiOo9JqDYz3qUKguJwW-7zceA4UtTr358R_CA/edit#heading=h.olhyszn0h7id